internal/integration/integration_test.go

```diff
@@ -493,6 +493,87 @@ func testCLISchemaApplyAutoApprove(t T, h string, dsn string, args ...string) {
 	require.NotNil(t, u)
 }
 
+func testCLISchemaApplyFromMigrationDir(t T) {
+	const (
+		DB    = "apply_migration_dir"
+		DevDB = DB + "_dev"
+	)
```

Comment:
keep them lowercased. 
```suggestion
		dbname    = "apply_migration_dir"
		devname = dbname + "_dev"
	)
```